third_party/stream-tests/Makefile

```diff
@@ -1,23 +1,42 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT license.
 
-CC= /opt/AMD/aocc-compiler-4.0.0/bin/clang
-CFLAGS= -Ofast -mcmodel=large -mavx2 -ffp-contract=fast -lomp -fopenmp -fnt-store=aggressive -DNTIMES=10
-GENFLAGS= -DSTREAM_ARRAY_SIZE=400000000 
-ZEN3FLAGS= -DSTREAM_ARRAY_SIZE=400000000 -march=znver3 
-ZEN4FLAGS= -DSTREAM_ARRAY_SIZE=800000000 -march=znver4 
+GENFLAGS= -DSTREAM_ARRAY_SIZE=400000000
+ZEN3FLAGS= -DSTREAM_ARRAY_SIZE=400000000 -march=znver3
+ZEN4FLAGS= -DSTREAM_ARRAY_SIZE=800000000 -march=znver4
+NEO2FLAGS= -DSTREAM_ARRAY_SIZE=120000000 -mcpu=neoverse-v2
+
 GEN_OUTPUT= streamx86.exe
 ZEN3_OUTPUT= streamZen3.exe
 ZEN4_OUTPUT= streamZen4.exe
+NEO2_OUTPUT= streamNeo2.exe
+
+ARCH := $(shell uname -m)
 
+ifeq ($(ARCH), aarch64)
+CFLAGS = -Ofast -fopenmp -DNTIMES=200
+CC=gcc
+all: NEO2
+else
+CC= /opt/AMD/aocc-compiler-4.0.0/bin/clang
```

Comment:
pls unify the spaces arounds `=`:
```
CFLAGS = xx
CC=xx
CC= xx
```

Reply:
made changes for consistency across the file.